File: src/oldmaid/Table.java
package oldmaid;

import java.util.ArrayList;

public class Table {
    private ArrayList<Card[]> disposedCards_ = new ArrayList<Card[]>();

    public void disposeCard(Card card[]) {
        for (int index = 0; index < card.length; index++) {
            System.out.print(card[index] + " ");
        }

        System.out.println("を捨てました");
        disposedCards_.add(card);
    }
}

File: python-oop/src/oldmaid/table.py
class Table():

    def __init__(self):
        self.disposed_cards = []

    # カードを捨てる。
    def dispose_card(self,dispose_list):
        for card in dispose_list:
            print(f"{card.toString()} を捨てました")
            self.disposed_cards.append(card)

    # 捨てられたカードを文字列で表現する。
    def toString(self):
        disposed_cards = ""
        for card in self.disposed_cards:
            disposed_cards += card.toString() + " "
        return disposed_cards
